performancetest/src/main/java/org/corfudb/performancetest/CorfuTablePerformanceTest.java

```diff
@@ -0,0 +1,151 @@
+package org.corfudb.performancetest;
+
+import com.google.common.reflect.TypeToken;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.runtime.CorfuRuntime;
+
+import org.corfudb.runtime.collections.CorfuTable;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.*;
+
+/**
+ * Created by Lin Dong on 10/18/19.
+ */
+
+@Slf4j
+public class CorfuTablePerformanceTest {
+    private String endPoint = "localhost:9000";
+    private int metricsPort = 1000;
+    private long seed = 1024;
+    private int randomBoundary = 100;
+    private long time = 1000;
+    private int milliToSecond = 1000;
+    private int keyNum = 10;
+    private int valueSize = 1024;
+    private CorfuRuntime runtime;
+    private Random random;
+    private static final Properties PROPERTIES = new Properties();
+    private KeyValueManager keyValueManager;
+
+    public CorfuTablePerformanceTest() {
+        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+        InputStream input = classLoader.getResourceAsStream("PerformanceTest.properties");
+        try {
+            PROPERTIES.load(input);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        loadProperties();
+        random = new Random(seed);
+        keyValueManager = new org.corfudb.performancetest.KeyValueManager(keyNum, valueSize);
+    }
+
+    private void loadProperties()  {
+        metricsPort = Integer.parseInt(PROPERTIES.getProperty("tableMetricsPort", "1000"));
+        endPoint = PROPERTIES.getProperty("endPoint", "localhost:9000");
+        seed = Long.parseLong(PROPERTIES.getProperty("tableSeed", "1024"));
+        randomBoundary = Integer.parseInt(PROPERTIES.getProperty("tableRandomBoundary","10"));
+        time = Long.parseLong(PROPERTIES.getProperty("tableTime", "100"));
+        milliToSecond = Integer.parseInt(PROPERTIES.getProperty("milliToSecond", "1000"));
+        keyNum = Integer.parseInt(PROPERTIES.getProperty("keyNum", "10"));
+        valueSize = Integer.parseInt(PROPERTIES.getProperty("valueSize", "1024"));
+    }
+
+    private CorfuRuntime initRuntime() {
+        CorfuRuntime.CorfuRuntimeParameters parameters = CorfuRuntime.CorfuRuntimeParameters.builder().build();
+        parameters.setPrometheusMetricsPort(metricsPort);
+        CorfuRuntime corfuRuntime = CorfuRuntime.fromParameters(parameters);
+        corfuRuntime.addLayoutServer(endPoint);
+        corfuRuntime.connect();
+        return corfuRuntime;
+    }
```

Comment:
use an abstract class to remove duplicate functions.

Reply:
good point